modules/custom/core/govcms_media/config/install/entity_browser.browser.govcms_media_editor_browser.yml

```
@@ -37,12 +37,13 @@ widgets:
     weight: -8
     label: 'Embed code'
     id: embed_code
-  ad2f2928-14d2-434a-8b70-3efbc71dced0:
+  d611eac6-3c64-4343-8ac9-4b2068cfe687:
     settings:
-      submit_text: Place
-      form_mode: summary
-      return_file: 0
-    uuid: ad2f2928-14d2-434a-8b70-3efbc71dced0
+      upload_location: 'public://'
+      multiple: true
+      submit_text: 'Select files'
+      extensions: 'jpg jpeg gif png txt doc xls pdf ppt pps odt ods odp'
```

should we add the standard office *x extensions here too for completeness?
```suggestion
      extensions: 'jpg jpeg gif png txt doc docx xls xlsx pdf ppt pptx pps ppsx odt ods odp'
```

---

yes as they are the newer, more standards compliant, and more widely used versions.
You should also consider svg though iirc that isn't always liked by drupal sites unless done in theme.

I'd be tempted by adding zip and the template versions (dotx etc) but they'd only be needed by some sites and can add that to their own config.

---

We had a request to also allow office files with macros (.xlsm). Should these be added/allowed, or treated same as executables (require them to be zipped. and allow .zip extensions instead)?

---

I'd be inclined to allow them on a site by site basis similar to zip and what I suggested with the template versions. I don't think they would need to be zipped.